modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/creole/creole_parser.js

```diff
@@ -343,12 +343,12 @@
 
 					var img = document.createElement('img');
 					img.src = imagePath;
-					img.alt =
-						r[2] === undefined
-							? options && options.defaultImageText
-								? options.defaultImageText
-								: ''
-							: r[2].replace(/~(.)/g, '$1');
+					if(r[2] != undefined){
```

Comment:
wouldn't `if (r[2]) {` be enough?

Reply:
I'm not a javascript guru, I left the previous condition, just negated it.